samples/tests: flaky gocdk-pubsub test due to order of stdout/stderr

```
=== RUN   Test
--- FAIL: Test (0.04s)
    main_test.go:45: pubsub.ct:23: got=-, want=+
        $ gocdk-pubsub sub -n 1 rabbit://sample-subscription
          []string{
        - 	"this is the message",
        + 	`Receiving messages from "rabbit://sample-subscription"...`,
        - 	`Receiving messages from "rabbit://sample-subscription"...`,
        + 	"this is the message",
          }
        
        
fail gocloud.dev/samples/gocdk-pubsub/Test (0.04s)
2019/07/10 00:16:34 working around #33016, which put us in /home/travis/gopath/src/gocloud.dev/samples/gocdk-pubsub
FAIL
coverage: 3.5% of statements in ./...
FAIL	gocloud.dev/samples/gocdk-pubsub	0.129s
fail gocloud.dev/samples/gocdk-pubsub (0.13s)
pass gocloud.dev/samples/order (28.91s)
Failures (reporting up to 10):
  gocloud.dev/samples/gocdk-pubsub/Test
ran 18; passed 6; failed 1; skipped 11 (in 55.8 sec)
```

--- a/samples/gocdk-pubsub/main.go
+++ b/samples/gocdk-pubsub/main.go
@@ -145,14 +145,14 @@ func (cmd *subCmd) Execute(ctx context.Context, f *flag.FlagSet, _ ...interface{
 	defer sub.Shutdown(ctx)
 
 	// Receive messages from the subscription and print them to stdout.
-	fmt.Fprintf(os.Stderr, "Receiving messages from %q...\n", subURL)
+	fmt.Printf("Receiving messages from %q...\n", subURL)
 	for i := 0; cmd.n == 0 || i < cmd.n; i++ {
 		m, err := sub.Receive(ctx)
 		if err != nil {
 			log.Print(err)
 			return subcommands.ExitFailure
 		}
-		fmt.Fprintf(os.Stdout, "%s\n", m.Body)
+		fmt.Printf("%s\n", m.Body)
 		m.Ack()
 	}
 	return subcommands.ExitSuccess

